src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -1097,26 +1094,19 @@ void ExpressAsCashFlows(EnergyPlusData &state)
     int firstMonth;
     int monthsBaseToService;
 
-    std::map<int, std::map<DataGlobalConstants::ResourceType, Real64>> resourceCosts;
+    std::map<int, std::array<Real64, static_cast<int>(Constant::ResourceType::Num)>> resourceCosts;
     for (int jMonth = 1; jMonth <= 12; ++jMonth) {
-        std::map<DataGlobalConstants::ResourceType, Real64> monthMap;
-        for (ResourceType iResource : state.dataGlobalConst->AllResourceTypes) {
-            monthMap.insert(std::pair<DataGlobalConstants::ResourceType, Real64>(iResource, 0.0));
-        }
-        resourceCosts.insert(std::pair<int, std::map<DataGlobalConstants::ResourceType, Real64>>(jMonth, monthMap));
+        resourceCosts[jMonth] = std::array<Real64, static_cast<int>(Constant::ResourceType::Num)>();
```

Comment:
@rraustad, this is the fix for the local `std::array` that looked strange to you. Each month does need its own array so that they don't overwrite each other, but you are right that you don't need to construct a local one and then copy construct it over. That was an extra copy which is now gone.